src/advanced_inv.cpp

```diff
@@ -2170,12 +2210,24 @@ void advanced_inventory::draw_minimap()
             invert_color(c_ltcyan) : c_ltcyan | A_BLINK;
         mvwputch(minimap, pt.y, pt.x, static_cast<nc_color>(cl), sym);
     }
-    // draw the player iff neither pane's area isn't in `places'
-    bool lip = is_any_of(panes[src].get_area(), places);
-    bool rip = is_any_of(panes[dest].get_area(), places);
-    if(!(lip && rip)) {
-        // draw the player in the center of the map
-        g->u.draw(minimap, g->u.pos(), (lip || rip));
+    /* I now present to you, a story of killer moves and even chiller grooves */
```

Comment:
I'm going to have to be a curmudgeon here and say I can't figure out what this is doing because of the variable names.